query/src/main/java/org/infinispan/query/impl/massindex/IndexUpdater.java

```diff
@@ -62,6 +62,7 @@ public void updateIndex(Object key, Object value) {
             DocumentBuilderIndexedEntity docBuilder = entityIndexBinding.getDocumentBuilder();
             final String idInString = keyTransformationHandler.keyToString(key);
             UpdateLuceneWork updateTask = docBuilder.createUpdateWork(
+                  null,
```

Comment:
- right we don't have tenant ids in Infinispan (maybe in future we'll have?)
- we do change SPIs in minor releases. You think it was worth overloading? We can still improve that before the final.